src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+Not shown to the virologist would be wether or not the virus is capable of jumping species type, nor the major health effect related to such an event occurring. 
+
+## Viral Mutations
+Viruses on their own will all start with a benign type symptom, or, the carrier of the virus won't know it exists. Over time, viruses will pass between crew naturally through normal work, physical contact, being near each other, and potential medical malpractice with improperly sterilized tools. 
+As the viruses transfer between hosts, there would be a variable chance for the virus to mutate based on the volatility rating of the virus (min of about 10% and max of 75%). When the virus mutates, it can select from a (hopefully) large list of symptoms and maladies, from literally nothing, to a cough or fever, to a series of more lethal traits like coughing blood, blindness, ecetra.
+The point buy can also remove previous traits the virus had, at a moderate weighting.
+Viral volatility should reroll every time the virus mutates. It should be possible for there to be a novel virus that only affects one or three crewmembers before it changes to a different virus based on the original.
+
+## Combating Viruses
+Cleanliness will go a long way to preventing viruses from getting out of hand. Medical tools, such as syringes will have a high chance to pick up viruses and transfer them between patients if the syringes are not sterilized properly. This can be done via a sterilization machine which would need to be added inside of medical, or in a pinch, you can use liquid soaps or space cleaners to attempt to clean syringes if you're in a rush, but with a much reduced probability of success.
+Alternatively, virologists can attempt to create vaccines to the virus in question, in order to create a vaccine, the virologist would need to provide multiple samples of the virus to the vaccinatior machine, which will then synthesize a vaccine with a hidden success probability based on the volatility of the target virus. Vaccine production will require *something* to actually create the vaccine. 
+A successful vaccine will prevent the specific virus the vaccine is created for from jumping to any person who has not already been infected by the vaccine parent, but will not prevent any future mutation of the same virus eventually coming around and infecting later.
+
+## External Infection Sources
+Viruses that don't originate in the crew round start can be instead picked up from other sources, such as raw meats in the kitchen, not cleaning meat cutting tools in the kitchen, rats and mice, artifacts creating (usually very lethal) diseases, and sometimes external sources, such as carp bites, vent spider bites, spawned blood puddles, and potentially bio terror from nukkies.
```

Comment:
I agree that there should be a small and inconsistent but ongoing supply of new pathogens to the station, otherwise virologists will probably get bored pretty quick.

Maybe we could have salvage wrecks spawn infected items or mobs, or rarely have cargo orders come contaminated with a new virus? It makes sense for cargo to be one of the main vectors for new infections, after all that's how new stuff gets on station. And if it gives salvage another reason to play on station (getting checked for new diseases) that'd be a nice bonus.

I can also think of many other potential places where new viruses would make sense to spawn that could increase inter-departmental interaction, for example botany mutations, meat anomalies, infected meteors crashing into the station (and an engineer getting infected)...